includes/frontend.php

```diff
@@ -659,7 +659,7 @@ public function enqueue_styles() {
 
 			wp_enqueue_style( 'elementor-frontend' );
 
-			wp_enqueue_style( 'swiper' );
+//			wp_enqueue_style( 'swiper' );
```

Comment:
I have tested this with the enqueue commented out, and the test results are very good.